descriptors/ModuleDescriptor-template.json

```diff
@@ -642,6 +668,7 @@
             "scheduled-notice-storage.scheduled-notices.collection.get",
             "scheduled-notice-storage.scheduled-notices.item.delete",
             "scheduled-notice-storage.scheduled-notices.item.put",
+            "scheduled-notice-storage.scheduled-notices.item.post",
```

Comment:
Looked through the code and didn't find where this permission is used. How was missing of this dependency discovered?

Reply:
Thanks for checking, @dmytrotur. @mreno-EBSCO also doubled checked and found that this permission was indeed required for requests and not for any scheduled task. I removed it from the scheduled tasks' path patterns.